### PR TITLE
ENH: Separate surfaces and morphometrics into standalone outputs

### PR DESCRIPTION
**Pull request by mgxd**
Rather than handling the paring down of surfaces when needed, this PR adds an aggregator node to collect and sort each surface output into a L/R list, simplifying retrieval for future use.

### CODE REVIEW

**Review by effigies — APPROVED** [review]
This looks reasonable. Assume you've tested? I'm not sure if this gets exercised by CI.

**Comment by mgxd** [comment]
Nope, though looks like it at least ran & completed in the circle tests. I'm using it for #358 so will test be tested there

**Comment by mgxd** [comment]
Tested locally 👍

File: smriprep/workflows/surfaces.py [hunk 1]
```diff
@@ -294,7 +301,15 @@ def init_surface_recon_wf(*, omp_nthreads, hires, name="surface_recon_wf"):
         (autorecon_resume_wf, outputnode, [('outputnode.subjects_dir', 'subjects_dir'),
                                            ('outputnode.subject_id', 'subject_id')]),
         (gifti_surface_wf, outputnode, [('outputnode.surfaces', 'surfaces'),
-                                        ('outputnode.morphometrics', 'morphometrics')]),
+                                        ('outputnode.morphometrics', 'morphometrics'),
+                                        ('outputnode.midthickness', 'midthickness'),
+                                        ('outputnode.pial', 'pial'),
+                                        ('outputnode.white', 'white'),
+                                        ('outputnode.inflated', 'inflated'),
+                                        ('outputnode.morphometrics', 'morphometrics'),
```
Comment (effigies) [inline smriprep/workflows/surfaces.py:309]: Duplicate of L304.

```suggestion
```

**Comment by effigies** [comment]
I'm okay with merging this, but you left it in draft state. Good to go?

**Pull request merged**